server/ladders.ts

```diff
@@ -179,6 +197,13 @@ class Ladder extends LadderStore {
 			return null;
 		}
 
+		if (Dex.formats.get(this.formatid).gameType === 'multi' && !noPartner) {
```

Comment:
```suggestion
		if (Dex.formats.get(this.formatid).gameType === 'multi' && !partner) {
```
`!noPartner` would sound like it'd equate to `true` as in you do have a partner.